firmware/lucidgloves-firmware/src/Main.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <mutex>
+#include "Controller/Gesture.h"
+#include "Controller/Haptics.h"
+#include "Encoding/AlphaEncoding.h"
+#include "Encoding/LegacyEncoding.h"
+#include "Communication/SerialCommunication.h"
+#include "Communication/BTSerialCommunication.h"
+#include "../AdvancedConfig.h"
```

Comment:
Not sure how arduino does its include directories, can this just be(?):
```suggestion
#include "AdvancedConfig.h"
```

Reply:
This causes an include error, so it seems the full relative path needs to be provided.

``fatal error: AdvancedConfig.h: No such file or directory
#include "AdvancedConfig.h"``